src/rrdp.rs

```diff
@@ -126,22 +149,24 @@ impl NotificationFile {
     ///
     /// Returns whether there are no gaps in the retained deltas.
     pub fn sort_and_verify_deltas(&mut self, limit: Option<usize>) -> bool {
```

Comment:
Looks like this method returns `true` whenever `self.deltas` is `Err(..)`. Is that the intention?

Reply:
My reasoning was that if there are no deltas, there are no gaps in them.

Reply:
Guess that makes sense though it is implicit and only holds for the 'Oversized` variant of `DeltaListError`. Which obviously works as it's the only variant for now.

Reply:
Yeah, it’s not perfect at all. I need to look at how Krill uses this and perhaps rework the whole approach.

Reply:
That makes sense, resolving for now then.